src/components/layout/welcome_banner.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { Header } from 'components/elements'
+import device from 'themes/device'
+import ukbanner from 'images/common/eu-uk-welcome-banner/uk_banner.png'
+import eubanner from 'images/common/eu-uk-welcome-banner/eu_banner.png'
+import eubannerMobile from 'images/common/eu-uk-welcome-banner/eu_mobile.png'
+import ukbannerMobile from 'images/common/eu-uk-welcome-banner/uk_mobile.png'
+import closeBanner from 'images/common/eu-uk-welcome-banner/close_icon.png'
+
+const countriesBanner = {
+    uk: ukbanner,
+    eu: eubanner,
+    euMobile: eubannerMobile,
+    ukMobile: ukbannerMobile,
+}
+
+const BannerWrapper = styled.section<{ country: string }>`
+    background-color: #3c77ae;
```

Comment:
Do we have this color in our theme? If not can we add this?

Reply:
okay, let me checkout

Reply:
No we don't have it actually.